internal/storage/rocksdb/store.go

```diff
@@ -250,6 +252,38 @@ func (rdb *rocksDB) Close() error {
 	return nil
 }
 
+func (rdb *rocksDB) ReplaceDB(checkpointDir string) error {
```

Comment:
Please convert this to private function.

Reply:
Done